static/app/app/list/views.js

```diff
@@ -59,8 +74,15 @@ var LunchMovesView = Marionette.CompositeView.extend({
     },
     template: _.template(LunchMovesTpl),
     childView: LunchMoveView,
+    childViewOptions: function(params) {
+      return { recentlySaved: this.recentlySaved };
+    },
     emptyView: EmptyView,
     childViewContainer: '.moves-container',
+    initialize:function(params) {
+        this.recentlySaved = params.recentlySaved;
+        this.childViewOptions();
```

Comment:
You don't have to call `childViewOptions` like this for it to work. If you make `childViewOptions` a function, Marionette automatically calls it when instantiating the child view instances.